cl/simple_pages/templates/help/recap_email_help.html

```diff
@@ -0,0 +1,95 @@
+{% extends "base.html" %}
+
+{% block title %}Help with @recap.email – CourtListener.com{% endblock %}
+{% block footer-scripts %}
+  {% include "includes/anchors.html" %}
+{% endblock %}
+{% block sidebar %}{% endblock %}
+
+
+{% block content %}
+<div class="hidden-xs col-sm-2 col-md-3"></div>
+<div class="col-xs-12 col-sm-8 col-md-6">
+    <h1>Help with @recap.email</h1>
+    <p class="lead">@recap.email is a system we use to gather content from PACER and add it to the <a href="https://www.courtlistener.com/recap/">RECAP Archive</a>. If you receive notification emails from PACER, it only takes a minute to set up this system and begin contributing content to the public commons.
+    </p>
+
+    <h2>How it works</h2>
+    <h3>Three easy steps:</h3>
```

Comment:
I think this would be better, but a screenshot would help me be sure.

```suggestion
    <p>Three easy steps:</p>
```